keyboards/lattice60/config.h

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 Ethan Durrant (emdarcher)
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x16c0
+#define PRODUCT_ID      0x27db
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    emdarcher
+#define PRODUCT         LATTICE60
+#define DESCRIPTION     A HHKB layout keyboard using only through-hole components.
```

Comment:
This value isn't used, IIRC.

```suggestion
```

Reply:
Even if it isn't used I can still keep it there right?

Reply:
Yeah, that's not a problem.